code/style.css

```diff
@@ -1,7 +1,431 @@
+  margin-bottom: 30px;
+  padding: 10px 40px 10px 40px;
+  font-size: 12px;
+  background-color: #E0D4FE;
+  opacity: 0.95;
+  position: relative;
+  z-index: 20;
+  border-radius: 2%;
+}
+
+label {
+  margin-bottom: 2%;
+  margin-top: 5%;
+}
+
+.blockLabel, .blockInput {
+  display: block;
+  
+}
+
+.radioGender {
+  display: inline-block;
+  padding: 2%;
+  margin-top: 4%;
+  margin-bottom: 2%;
+}
+
+#termsConditions, #newsletter {
+  padding: 2%;
+}
+
+.submitButton {
+  background-color: #FFF7AE;
+  padding: 5%;
+  margin-top: 5%;
+  margin-bottom: 5%;
+  border: solid;
+  border-width: 1px;
+  border-radius: 8%;
+  border-color: #9D98FA;
+  
+}
+
+/* ACCORDION CONTAINER */
+
+.accordionWrapper {
+  width: 80%;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  justify-content: center;
+}
+
+#accordion {
+  font-size: 12px;
+  margin-bottom: 10px;
+  padding: 10px 40px 30px 40px;
+  background-color: #E0D4FE;
+  opacity: 0.95;
+  position: relative;
+  z-index: 20;
+  border-radius: 5px;
+}
+
+.question {
+  background-color: #FFF7AE;
+  padding: 8%;
+  border:solid;
+  border-width: 1px;
+  border-color: #fcf08b;
+  font-weight: 600;
+}
+
+.answer {
+  line-height: 1.5;
+  background-color: #fffbdb;
+  border:solid;
+  border-width: 1px;
+  border-color: #fcf08b;
+  padding: 8%;
+}
+
+#question1, #question2, #question3, #question4 {
+  margin: 0;
+  min-width: 100%;
+}
+
+
+/* FOOTER */
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 20px;
+  padding: 5%;
+  position: relative;
+  z-index: 20;
+}
+
+.footerText {
+  display: flex;
+  font-family: "Permanent Marker";
+  font-size: 14px;
+  color: #FFF7AE;
+  margin-bottom: 8%;
+}
+
+.footerCred {
+  display: flex;
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  margin: 2px;
+  color: #FFF7AE;
+}
+
+
+/* --------!!! MEDIA QUERIES !!!-------------- */
+
+/* ------------- TABLET SIZE ----------- */
+
+@media (min-width: 667px) and (max-width: 1024px)  {
+
+
+  .headerText {
+    font-size: 80px;
+    margin-top: 8%;
+  }
+
+  
+ /* NAVBAR */
+
+li {
+  padding: 5%;
+}
+
+li a {
+  font-size: 26px;
+}
+
+  
+  /* MAIN CONTENT */
+
+  .flexParent {
+    display: flex;
+    flex-direction: row;
+    width: 80vw;
+    margin: auto;
+    margin-top: 2%;
+  }
+
+  .formWrapper {
+    width: 70vw;
+    margin-top: 3%;
+    margin-right: 3%;
+  }
+  
+  .accordionWrapper {
+    width: 70vw;
+    margin-top: 3%;
+    margin-left: 3%;
+  }
+
+  .question, .answer {
+    padding: 10%;
+  }
+
+  #question1, #question2, #question3, #question4 {
+    margin: 0;
+    width: 220px;
+  }
+
+  
+  /* FOOTER */
+  
+  .footer {
+    margin-top: 2%;
+    font-size: 24px;
+  }
+
+  .footerText {
+    font-size: 28px;
+    margin-bottom: 3%;
+  }
+  
+  .footerCred {
+    font-size: 14px;
+    margin: 2px;
+  }
+
+}
+
+
+/*----------------DESKTOP SIZE-------------------*/
+
+@media (min-width: 1024px) {
```

Comment:
Good job with the responsiveness of the desktop version! It looks like you had to try a lot of adjustments, but you nailed it in the end 🥇

Reply:
A LOT of adjustments 😅  thank you!!